README.md

```diff
@@ -247,6 +247,36 @@ found in both `node_modules` and `bower_components`.
 
 Now you should be ready to deploy.
 
+See also the note on [Javascript minification](#javascript-minification)
+
+## Javascript minification
+
+When precompiling assets in production, you will probably want to
+ensure that you are not minifying your javascript twice, in EmberCLI
+and in Rails. This can slow down assets precompilation considerably.
+
+There are two ways to do this. You can disable minification in Ember
+by modifying your `Brocfile.js` as follows:
+
+```javascript
+  var app = new EmberApp({
+    ...
+    minifyJS: false,
+    ...
+  }
+```
+
+or you can disable minification in Rails by setting:
+
+```ruby
+  config.assets.js_compressor = nil
+```
+
+in your `config/environments/production.rb` file. Disabling in Ember
+is the better route if it works for you, but it may be more fragile in
+the case of javascript errors.
```

Comment:
Your guess is as good as mine. :) I get this error in an app:

```
rake aborted!
ExecJS::ProgramError: Unexpected token: punc (:) (line: 1, col: 10, pos: 10)

Error
    at new JS_Parse_Error (/tmp/execjs20150321-24731-1hyu7atjs:2359:10623)
    at js_error (/tmp/execjs20150321-24731-1hyu7atjs:2359:10842)
    at croak (/tmp/execjs20150321-24731-1hyu7atjs:2359:19086)
    at token_error (/tmp/execjs20150321-24731-1hyu7atjs:2359:19223)
    at unexpected (/tmp/execjs20150321-24731-1hyu7atjs:2359:19311)
    at semicolon (/tmp/execjs20150321-24731-1hyu7atjs:2359:19784)
    at simple_statement (/tmp/execjs20150321-24731-1hyu7atjs:2359:22580)
    at /tmp/execjs20150321-24731-1hyu7atjs:2359:20274
    at /tmp/execjs20150321-24731-1hyu7atjs:2359:19957
    at block_ (/tmp/execjs20150321-24731-1hyu7atjs:2359:24599)
  (in /home/egh/c/tahi/tmp/ember-cli-cd98e71c-ad5a-4b81-85de-a24baaf26e75/assets/tahi/ember-data.js.map)new JS_Parse_Error ((execjs):2359:10623)
```

when I disable minifyJS in ember and use uglify in rails. I'll try to track it down, but it's pretty unclear to me.